Src/WitsmlExplorer.Frontend/services/credentialsService.ts

```diff
@@ -10,6 +10,11 @@ export interface BasicServerCredentials {
   password?: string;
 }
 
+type CookieInfo = {
+  type: "WExSession" | "WExPersistent" | "WExInvalid";
+  url: string;
```

Comment:
Why is the url part of the value if it is already the key?

Reply:
It was not trivial to access the key when fetching all relevant items in localstorage, see `refreshLocalstorageSessions` 
It might be irrelevant if using the credentials list to access localstorage so I will test that next.

Reply:
Agreed to keep as is for now to be re-assessed in later PR WE-614